### PR TITLE
update translate spanish

### PR DESCRIPTION
**Pull request by Eignar17**

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -779,8 +779,7 @@ msgstr ""
 msgid ""
 "If enabled, Elements will toggle with keys. Otherwise, it will toggle by "
 "checkboxes."
-msgstr "Si está habilitado, Elementos alternará con las teclas. De lo contrario, se activará"
-"Las casillas de verificación".
+msgstr ""
```
Comment (soreau) [inline po/es.po]: Why was this message removed?

Comment (Eignar17) [reply po/es.po]: bug compile

Comment (soreau) [reply po/es.po]: Ok

**Comment by Eignar17** [comment]
@rbuj please update translate

**Pull request merged**